### PR TITLE
SFR-1828_APIS3ObjectLink

### PR DESCRIPTION
**Pull request by mitri-slory**
This PR adds a new API that returns the a unique presigned url for S3 objects from the DRB buckets on AWS. The first commit I made to this PR has changes to the API and tests that are outdated now which I explain in the comments below. However, the first PR includes changes I made to swagger and other files which I recommend to look over as well for feedback.

### CODE REVIEW

File: api/blueprints/drbS3.py
Comment (mitri-slory) [inline api/blueprints/drbS3.py:1]: This API was the initial format where the link returned was the object url to the S3 object, but I realized later on that we needed to return a unique link to users so it the url won't be shared amongst users outside the organization.

File: tests/unit/test_api_s3_blueprint.py
Comment (mitri-slory) [inline tests/unit/test_api_s3_blueprint.py:1]: These tests were made with the initial return to an object url I had in mind for the S3 API, so this is outdated.

File: swagger.v4.json [hunk 1]
```diff
@@ -245,7 +245,7 @@
                     }
                 }
             }
-        },
+        },
```
Comment (Apophenia) [inline swagger.v4.json:248]:
```suggestion
        },
```

Comment (Apophenia) [reply swagger.v4.json:248]: If you can revert this whitespace change I think this will be ready to go!

**Review by Apophenia — CHANGES_REQUESTED** [review]
Logic is all good, not necessarily a merge blocker but if you could revert the whitespace change in the Swagger JSON that would be perfect.

**Review by Apophenia — APPROVED** [review]

**Pull request merged**